### PR TITLE
refactor(textarea): made necessary changes to current dev state

### PR DESCRIPTION
**Pull request by mfranzke**

### CODE REVIEW

File: packages/components/src/components/textarea/docs/Angular.md [hunk 1]
```diff
@@ -20,5 +20,5 @@ import { DBTextareaModule } from '@db-ui/ngx-components';
 
 ```html app.component.html
 <!-- app.component.html -->
-<db-textarea>Textarea</db-textarea>
+<db-textarea></db-textarea>
```
Comment (nmerget) [inline packages/components/src/components/textarea/docs/Angular.md:23]: I guess this is for the label, right?

File: packages/components/src/components/textarea/docs/React.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ For general installation and configuration look at the [react-components](https:
 // App.tsx
 import { DBTextarea } from "@db-ui/react-components";
 
-const App = () => <DBTextarea>Textarea</DBTextarea>;
+const App = () => <DBTextarea></DBTextarea>;
```
Comment (nmerget) [inline packages/components/src/components/textarea/docs/React.md:11]: https://github.com/db-ui/mono/pull/1386#discussion_r1289584180

File: packages/components/src/components/textarea/docs/Vue.md [hunk 3]
```diff
@@ -11,6 +11,6 @@ import { DBTextarea } from "@db-ui/v-components";
 </script>
 
 <template>
-	<DBTextarea>Textarea</DBTextarea>
+	<DBTextarea></DBTextarea>
```
Comment (nmerget) [inline packages/components/src/components/textarea/docs/Vue.md:14]: https://github.com/db-ui/mono/pull/1386#discussion_r1289584180

File: packages/components/src/components/textarea/textarea.lite.tsx [hunk 4]
```diff
@@ -56,11 +57,41 @@ export default function DBTextarea(props: DBTextareaProps) {
...
+			}
+
+			// TODO: Replace this with the solution out of https://github.com/BuilderIO/mitosis/issues/833 after this has been "solved"
+			// VUE:this.$emit("update:value", event.target.value);
+
+			// Angular: propagate change event to work with reactive and template driven forms
+			this.propagateChange(event.target.value);
+
 			// using controlled components for react forces us to use state for value
 			state._value = event.target.value;
```
Comment (nmerget) [inline packages/components/src/components/textarea/textarea.lite.tsx:74]: I guess we need to remove this `_value` like we did in the Input as well

File: showcases/patternhub/components/component-parser/index.tsx [hunk 5]
```diff
@@ -214,14 +214,6 @@ const ComponentSwitch = ({
 		);
 	}
 
-	if (type === 'textarea') {
```
Comment (nmerget) [inline showcases/patternhub/components/component-parser/index.tsx:217]: Why did you remove this?

Comment (mfranzke) [reply showcases/patternhub/components/component-parser/index.tsx:217]: Because it's included twice.

File: packages/components/src/components/textarea/docs/Angular.md (hunk 1 above)
Comment (mfranzke) [reply packages/components/src/components/textarea/docs/Angular.md:23]: No, the label is controlled via the `label`-attribute. The included content in here would be (if aligned with the regular `textarea` "API") the included initial value / "defaultValue". But I haven't been able to fix some problems that appeared on the way. Do you have the chance to give it another try?

Comment (nmerget) [reply packages/components/src/components/textarea/docs/Angular.md:23]: Ahh okay, would be nice if it works like this as well, right?

Can't you just pass the `props.children` into the `textarea`?

Comment (mfranzke) [reply packages/components/src/components/textarea/docs/Angular.md:23]: > Do you have the chance to give it another try, please?

this would be awesome, but sadly I haven't been able to do that. Do you have the chance to give it another try, please?
